code/style.css

```diff
@@ -1,4 +1,535 @@
+}
+
+.podcasts .item {
+  flex-grow: 1;
+  height: 3rem;
+  width: 20rem;
+  margin: .5rem .25rem;
+  display: flex;
+  overflow: hidden;
+  border-right: 1px solid grey;
+}
+
+.podcasts .item:last-child {
+  border: none;
+}
+
+.podcasts img {
+  width: 3rem;
+  border-radius: 50%;
+  margin-right: .5rem;
+  object-fit: cover;
+}
+
+.podcasts .text {
+  padding: 0 .5rem 0 0;
+}
+
+section .text, 
+.grid-inner .item  {
+  transition: .4s ease-out;
+  padding: .5rem;
+  padding-left: .75rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.story {
+  margin-top: 3rem;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.story div {
+  flex: 1;
+}
+
+.top-news .container {
+  transition: .1s ease-in-out;
+  display: flex;
+  flex-direction: column;
+  gap: .5rem;
+  margin-top: .5rem;
+}
+
+.top-news .container .text {
+  flex-grow: 2;
+}
+
+.top-news .container .image {
+  flex-grow: 4;
+}
+
+.top-news .container .optional {
+  display: none;
+  flex-grow: 1;
+}
+
+.full-length-news {
+  display: none;
+}
+
+.full-length-news div {
+  width: 100%;
+  height: 100%;
+  margin: .5rem 0;
+}
+
+.small-news {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: .5rem;
+}
+
+.grid-inner .item {
+  margin-top: 3rem;
+}
+
+footer {
+  margin: 0 -10rem;
+  padding: 1rem 11rem;
+  display: flex;
+  min-height: 8rem;
+  height: fit-content;
+  background-color: #222;
+  color: white;
+  margin-top: 5rem;
+  justify-content: space-between;
+}
+
+footer p {
+  width: 60%;
+}
+
+/* SMALL MOBILE */
+
+@media screen and (min-width: 375px) {
+
+  header > .middle {
+    width: 12rem;
+  }
+}
+
+/* TABLET */
+
+@media screen and (min-width: 667px) {
+
+  body {
+    max-width: 75rem;
+    padding: 0 1rem;
+  }
+
+  p {
+    font-family: 'Playfair Display', serif;
+    font-size: .8rem;
+    line-height: 1.5;
+    margin-top: .5rem;
+  }
+  
+   section .image {
+    width: 0;  
+    min-width: 10rem;
+  }
+
+  .story {
+    margin: 0;
+  }
+
+  header {
+    height: 3rem;
+    background-size: 16rem;
+    background-position: 50% 80%;
+  }
+
+  header > .middle {
+    width: 14rem;
+  }
+
+  header > .right {
+    flex-direction: column;
+  }
+  
+  .right > .hamburger {
+    display: none;
+  }
+  
+  .left > .hamburger,
+  header > .right > .subscribe,
+  .podcasts,
+  .full-length-news {
+    display: initial;
+  }
+
+  main .divider {
+    display: block;
+  }
+     
+  .top-news .container {
+    flex-direction: row;
+    max-height: 25rem;
+    height: 100vw;
+  }
+
+  .small-news {
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: flex-start;
+  }
+
+  .top-news .container .text {
+    width: 0;
+  }
+
+  .story {
+    flex: 1;
+    flex-direction: row;
+    min-width: 26rem;
+    max-height: 12rem;
+  }
+  
+  .grid-outer {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  
+  }
+  
+  .grid-inner .item {
+    border-top: 1px solid #EBEBEB;
+    border-left: 0;
+    margin-top: 2rem;
+  }
```

Comment:
What do you think about having the grid-outer as the only grid? I think it's OK like you have it too, just that the name grid-inner doesn't make sense since it's not a grid in mobile :) So to prevent that you could have just the grid-outer with some different rules for the different media queries

Reply:
My idea was to have grid-inner left and grid-inner right independent of each other, but i suppose i could have achieved that without doing a grid in a grid. Grid area maybe? Also - yes, this grid thing was very much not mbile first!